cmd/logs.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import "strconv"
+
+func init() {
+	register(&logsCmd{})
+}
+
+type logsCmd struct {
```

Comment:
this one command is sort of a utility knife command. depending on the args, it does everything we need with logs...